remediation/machine-deletion/how-it-works.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: How It Works
+parent: Machine Deletion
+grand_parent: Remediation
+nav_order: 1
+---
+
+# How Machine Deletion Remediation works?
+Machine Deletion Remediation (MDR) uses the [Machine API](https://github.com/openshift/machine-api-operator#readme) to reprovision unhealthy nodes.
+It was tested with [NodeHealthCheck](https://github.com/medik8s/node-healthcheck-operator#readme) (NHC), which creates a Custom Resource for MDR once a node turns unhealty.
+
+MDR follows the annotation on the Node to the associated Machine object, confirms that it has an owning controller (e.g. MachineSetController), and deletes it. Finally, the owning controller creates a replacement.
```

Comment:
Might worth explain what it means in this context.
IIRC checking for an owning controller is our way of verifying the node is a worker node and not a control-plane node.

Reply:
> IIRC checking for an owning controller is our way of verifying the node is a worker node and not a control-plane node

yes and no: this ensures that someone is recreating the machine when we delete it. Which usually is the case for worker machines on IPI clusters, AFAIK.